test/blink/2-keyframes-with-offsets.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
-<script src="testharness/testharness.js"></script>
-<script src="testharness/testharnessreport.js"></script>
+<script src="../resources/testharness.js"></script>
```

Comment:
Can you set this up as /resources or something now you have the 'proxies' config stuff?